release/release_version

```diff
@@ -0,0 +1 @@
+1.6.1-incubating
```

Comment:
nit: new line

Reply:
it's a config file.. no black line can safe our life